op-node/client/rpc.go

```diff
@@ -186,6 +187,13 @@ func (ic *InstrumentedRPCClient) EthSubscribe(ctx context.Context, channel any,
 	return ic.c.EthSubscribe(ctx, channel, args...)
 }
 
+func instrument1(m *metrics.Metrics, name string, cb func() error) error {
+	record := m.RecordRPCClientRequest(name)
+	err := cb()
+	record(err)
+	return err
+}
+
```

Comment:
left-over from `op-node/client/client.go` which dooesn't appear to be used at all